hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -1123,6 +1123,8 @@
   public static final String USE_META_REPLICAS = "hbase.meta.replicas.use";
   public static final boolean DEFAULT_USE_META_REPLICAS = false;
 
+  public static String LOCATOR_META_REPLICAS_MODE = "hbase.locator.meta.replicas.mode";
```

Comment:
Originally, I tried to avoid org.apache.hadoop.hbase.client import in org.apache.hadoop.hbase.ClientMetaTableAccessor. Checking again, there are client module imports in this file,
```
import org.apache.hadoop.hbase.client.Consistency;
import org.apache.hadoop.hbase.client.Get;

```
I will revert this change.

Reply:
Resolve it as the new update addresses the comment.